src/main/java/uk/ac/ic/wlgitbridge/snapshot/base/MissingRepositoryException.java

```diff
@@ -0,0 +1,43 @@
+package uk.ac.ic.wlgitbridge.snapshot.base;
+
+import com.google.gson.JsonElement;
+import uk.ac.ic.wlgitbridge.git.exception.SnapshotAPIException;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MissingRepositoryException extends SnapshotAPIException {
+
+    public static final String GENERIC_REASON =
+        "This Overleaf project currently has no git access.\n" +
+        "\n" +
```

Comment:
In other similar errors, it looks like we're returning `List<String>` of lines from `getDescriptionLines` rather than having the line breaks in directly. Possibly that's something to do with cross-platform compatibility? I can't actually remember why it does that rather than just having line breaks in the strings.